src/lib/components/atoms/EditContent.svelte

```diff
@@ -0,0 +1,18 @@
+<script type="ts">
+	import Icon from '@iconify/svelte';
+
+	export let href: string;
+	export let target: '_blank' | '_self' | '_parent' | '_top';
+	export let rowDistance: number;
+</script>
+
+<a {href} {target} rel="noopener noreferrer" class={`header-link row-${rowDistance} align-center`}>
+	<Icon icon="tabler:brand-github" />
+	Edit content
+</a>
+
+<style type="scss">
+	a {
+		margin-left: var(--space-1);
```

Comment:
Same here.
The margin this button has to its left is the responsibility of its parent.
I believe that not having this style would make the component more reusable.

Reply:
Same here, already fixed!